request.rest

```diff
@@ -0,0 +1,27 @@
+### Add a new comment
+POST http://localhost:3000/comment
+Content-Type: application/json
+
+{
+  "postId": "<post_id>",
```

Comment:
the `<post_id>` is an intersting way to write it, I did similar to Ron with variable placeholders, I wonder which is best

Reply:
becuase you added placeholder variables you should write {{post_id}} instead of <post_id>